posttroll/publisher.py

```diff
@@ -130,6 +134,18 @@ def __init__(self, address, name="", min_port=None, max_port=None):
         self._heartbeat = None
         self._pub_lock = Lock()
 
+    def _bind_destination(self):
+        """Bind publish destination."""
+        last_error = ""
+        for _ in range(BIND_RETRIES):
```

Comment:
This doesn't work if `BIND_RETRIES` is 0, which would be a logical value if you just want one try but no retries.

Reply:
A test should be added for that case btw.

Reply:
This was modified and there's a test that `BIND_RETRIES` is at least `0`.